github/orgs_security_managers_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestOrganizationsService_ListSecurityManagerTeams(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/orgs/o/security-managers", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `[{"id":1}]`)
+	})
+
+	ctx := context.Background()
+	teams, _, err := client.Organizations.ListSecurityManagerTeams(ctx, "o")
+	if err != nil {
+		t.Errorf("Organizations.ListSecurityManagerTeams returned error: %v", err)
+	}
+
+	want := []*Team{{ID: Int64(1)}}
+	if !cmp.Equal(teams, want) {
+		t.Errorf("Organizations.ListSecurityManagerTeams returned %+v, want %+v", teams, want)
+	}
+
+	const methodName = "ListSecurityManagerTeams"
+	testBadOptions(t, methodName, func() (err error) {
+		_, _, err = client.Organizations.ListSecurityManagerTeams(ctx, "\n")
+		return err
+	})
+
+	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
+		got, resp, err := client.Organizations.ListSecurityManagerTeams(ctx, "o")
+		if got != nil {
+			t.Errorf("testNewRequestAndDoFailure %v = %#v, want nil", methodName, got)
+		}
+		return resp, err
+	})
+}
+func TestOrganizationsService_ListSecurityManagerTeams_invalidOrg(t *testing.T) {
```

Comment:
Please add a blank line above this.  I'm surprised `gofmt` didn't catch this.
Please make sure to run `gofmt` as described in CONTRIBUTING.md.

Reply:
Yeah, weird. Ran it multiple times to check. 🤷 

Newline added.